### ISSUE TITLE
Cannot import mongoose

### ISSUE THREAD

**Issue by ESnark** [lukehaas/RunJS#197]
I got this message while I'm trying to import mongoose
`TypeError: Cannot assign to read only property 'Symbol(nodejs.util.inspect.custom)' of object '[object Object]'`

```javascript
require('mongoose')
// or
import { Schema } from 'mongoose'
new Schema()
```

**Comment by lukehaas** [comment]
@ESnark thanks for raising this. This will be resolved in the next release.

**Comment by lukehaas** [comment]
@ESnark this has now been resolved in version 1.13.0
